packages/openjphjs/src/HTJ2KEncoder.hpp

```diff
@@ -29,11 +30,16 @@ class HTJ2KEncoder
   /// Constructor for encoding a HTJ2K image from JavaScript.
   /// </summary>
   HTJ2KEncoder() : decompositions_(5),
+                   request_tlm_marker_(false),
+                   set_tilepart_divisions_at_resolutions_(false),
+                   set_tilepart_divisions_at_components_(false),
                    lossless_(true),
                    quantizationStep_(-1.0),
                    progressionOrder_(2), // RPCL
                    blockDimensions_(64, 64)
   {
+    // Use for debugging to ensure that correct encoder is being used
+    OJPH_INFO(0x00010001, "Hello HTJ2K Encoder - v1");
```

Comment:
can we remove this?

Reply:
Commented out, but it is really useful to have as commented code to test reload conditions - it isn't obvious when you get a reload necessarily.